monitoring/nethealth.go

```diff
@@ -303,7 +303,6 @@ func (c *nethealthChecker) fetchNethealthMetrics(ctx context.Context) (res []byt
 				return net.Dial("unix", c.NethealthSocketPath)
 			},
 		},
-		Timeout: time.Second,
```

Comment:
Not sure why this is removed - usually, it makes sense to limit the http client to avoid unrestricted connect attempts. So using either timeouts or context would be preferred.